moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -1807,8 +1807,12 @@ bool PlanningScene::processCollisionObjectRemove(const moveit_msgs::CollisionObj
   else
   {
     world_->removeObject(object.id);
-    removeObjectColor(object.id);
-    removeObjectType(object.id);
+    if (!robot_state_->hasAttachedBody(object.id))
```

Comment:
This should fix the segfault in CI. Not every scene has a robot_state_ itself.

```suggestion
    if (!getCurrentState()->hasAttachedBody(object.id))
```

My other point remains though, I don't think this this patch makes sense.

Reply:
right!!!   thanks!